cmd/flipt/main.go

```diff
@@ -88,15 +89,35 @@ var (
 )
 
 func main() {
+	var (
+		once         sync.Once
+		loggerConfig = zap.Config{
+			Level:            zap.NewAtomicLevelAt(zap.InfoLevel),
+			Development:      true,
+			Encoding:         "console",
+			EncoderConfig:    zap.NewDevelopmentEncoderConfig(),
```

Comment:
like you mentioned in the PR description, it might be worth us looking into creating our own `EncoderConfig`, for instance to allow formatting the `timestamps` to use something like RFC3339 instead of the full output like `2022-09-10T09:49:40.178-0400`

re: https://github.com/uber-go/zap/issues/485#issuecomment-820995909

Reply:
+1 on RFC3339